lib/email/utils/helpers.js

```diff
@@ -28,7 +30,7 @@ function logFlowEventFromMessage(log, message, type) {
   const elapsedTime = currentTime - flowBeginTime
 
   if (flowId && flowBeginTime && (elapsedTime > 0) && type && templateName) {
-    const eventName = `email.${templateName}.${type}`
+    const eventName = `email.${templateName}.${type}.${emailDomain}`
```

Comment:
> Will changing this event break any existing dashboards?

Moving forward it would I think, however, we could archive (take a picture?) old dashboard and then update to use new query.

Possible query to get all email bounces
```
  funnel_event_flags AS (
    SELECT
      f.day,
      f.flow_id,
      COUNT(CASE WHEN e.type LIKE 'email.%.bounced%' THEN true ELSE NULL END) > 0 AS did_email_bounced,
      COUNT(CASE WHEN e.type LIKE 'email.%.delivered%' THEN true ELSE NULL END) > 0 AS did_email_delivered
    FROM flows AS f
    INNER JOIN flow_events AS e
    ON f.flow_id = e.flow_id AND e.timestamp::DATE >= f.day AND e.timestamp::DATE <= f.day + '1 day'::INTERVAL
    GROUP BY 1, 2
    ORDER BY 1, 2
  ),
```
Possible sample query to get bounced, delivered and total for all templates sent to aol.
```
  funnel_event_flags AS (
    SELECT
      f.day,
      f.flow_id,
      COUNT(CASE WHEN e.type LIKE 'email.%.bounced.aol.com' THEN true ELSE NULL END) > 0 AS did_email_bounced,
      COUNT(CASE WHEN e.type LIKE 'email.%.delivered.aol.com' THEN true ELSE NULL END) > 0 AS did_email_delivered
    FROM flows AS f
    INNER JOIN flow_events AS e
    ON f.flow_id = e.flow_id AND e.timestamp::DATE >= f.day AND e.timestamp::DATE <= f.day + '1 day'::INTERVAL
    GROUP BY 1, 2
    ORDER BY 1, 2
  ),
```
A downfall/benefit I can see, is that we probably shouldn't add more values after appending domain. Or maybe we use a different delimeter than `.` because domains include it.